tmt/export.py

```diff
@@ -513,15 +604,173 @@ def export_to_nitrate(test):
+    test.tag.append('fmf-export')
+    if not dry_mode:
+        polarion_case.tags = ' '.join(test.tag)
+    echo(style('tags: ', fg='green') + ' '.join(set(test.tag)))
+
+    # Default tester
+    if test.contact:
+        # Need to pick one value, so picking the first contact
+        email_address = email.utils.parseaddr(test.contact[0])[1]
+        login_name = email_address[:email_address.find('@')]
         try:
             if not dry_mode:
-                bz_set_coverage(
-                    bz_instance, verifies_bug_ids, int(
-                        nitrate_case.id))
-            echo(style("Linked to Bugzilla: {}.".format(", ".join(
-                [f"BZ#{bz_id}" for bz_id in verifies_bug_ids])), fg='magenta'))
+                polarion_case.add_assignee(login_name)
+            echo(style('default tester: ', fg='green') + login_name)
+        except PolarionException as err:
+            log.debug(err)
+
+    # Status
+    if not dry_mode:
+        if test.enabled:
+            polarion_case.status = 'approved'
+        else:
+            polarion_case.status = 'inactive'
+    echo(style('enabled: ', fg='green') + str(test.enabled))
+
+    echo(style("Append the Polarion test case link.", fg='green'))
+    if not dry_mode:
+        with test.node as data:
+            server_url = str(polarion_case._session._server.url)
+            add_link(
+                f'{server_url}{"" if server_url.endswith("/") else "/"}'
+                f'#/project/{polarion_case.project_id}/workitem?id='
+                f'{str(polarion_case.work_item_id)}',
+                data, system=SYSTEM_OTHER, type_='implements')
+
+    # List of bugs test verifies
+    bug_ids = []
+    requirements = []
+    for link in test.link:
+        try:
+            bug_ids.append(
+                re.search(RE_BUGZILLA_URL, link['verifies']).group(1))
         except Exception as err:
-            raise ConvertError("Couldn't update bugs", original=err)
+            log.debug(err)
+        try:
+            requirements.append(
+                re.search(RE_POLARION_URL, link['verifies']).group(1))
+        except Exception as err:
+            log.debug(err)
+
+    # Add bugs to the Polarion case
+    if not dry_mode:
+        polarion_case.tcmsbug = ', '.join(bug_ids)
+
+    # Add TCMS Case ID to Polarion case
+    if test.node.get('extra-nitrate') and not dry_mode:
+        polarion_case.tcmscaseid = str(int(
+            re.search(r'\d+', test.node.get("extra-nitrate")).group()))
+
+    # Add uuid to Polarion case
+    uuid = add_uuid_if_not_defined(test.node, dry=dry_mode)
+    if not uuid:
+        uuid = test.node.get(ID_KEY)
+    if not dry_mode:
+        polarion_case.test_case_id = uuid
+    echo(style(f"Append the UUID {uuid}.", fg='green'))
+
+    # Add Requirements to Polarion case
+    if not dry_mode:
+        for req in requirements:
+            polarion_case.add_linked_item(req, 'verifies')
+
+    # Update Polarion test case
+    if not dry_mode:
```

Comment:
Are the `dry` mode checks above needed if the `polarion_case.update()` is under the condition here?

Reply:
yeah, polarion_case is None if dry mode is enabled, so exceptions would happen